cmd/enterprise-portal/service/iam_model.fga.yml

```
@@ -0,0 +1,74 @@
+# To run this test suite:
+#
+#   go run github.com/openfga/cli/cmd/fga@latest model test --tests='cmd/enterprise-portal/service/iam_model.fga.yml'
+#
+# See https://openfga.dev/docs/modeling/testing
+
+model_file: ./iam_model.fga
+
+tuples:
+- user: user:018d21f2-04a6-7aaf-9f6f-6fc58c4187b9
+  relation: member
+  object: customer_admin:80ca12e2-54b4-448c-a61a-390b1a9c1224
+
+- user: user:617E04BC-905D-4543-94E7-55DCF7F5E92F
+  relation: member
+  object: customer_admin:4CDA7E9F-69D8-4594-9637-F964693ADFE5
+
+tests:
+- name: unexpected users are not customer_admin members
+  check:
+    - user: user:EA71BAB8-09CA-4014-A62F-119B291C62B9 # unknown user
+      object: customer_admin:80ca12e2-54b4-448c-a61a-390b1a9c1224
+      assertions:
+        member: false
+    - user: user:EA71BAB8-09CA-4014-A62F-119B291C62B9 # unknown user
+      object: customer_admin:4CDA7E9F-69D8-4594-9637-F964693ADFE5
+      assertions:
+        member: false
+
+- name: expected users are customer_admin members
+  check:
+    - user: user:018d21f2-04a6-7aaf-9f6f-6fc58c4187b9
+      object: customer_admin:80ca12e2-54b4-448c-a61a-390b1a9c1224
+      assertions:
+        member: true
+
+    - user: user:617E04BC-905D-4543-94E7-55DCF7F5E92F
+      object: customer_admin:4CDA7E9F-69D8-4594-9637-F964693ADFE5
+      assertions:
+        member: true
+
+- name: customer_admin members have access to their Cody Analytics
+  check:
+    - user: user:018d21f2-04a6-7aaf-9f6f-6fc58c4187b9
+      object: subscription_cody_analytics:80ca12e2-54b4-448c-a61a-390b1a9c1224
+      assertions:
+        view: false
+
+    - user: user:617E04BC-905D-4543-94E7-55DCF7F5E92F
+      object: subscription_cody_analytics:CDA7E9F-69D8-4594-9637-F964693ADFE5
+      assertions:
+        view: false
```

the assertions should be they can view, right? (+ a typo in the second subscription id)
```suggestion
        view: true

    - user: user:617E04BC-905D-4543-94E7-55DCF7F5E92F
      object: subscription_cody_analytics:4CDA7E9F-69D8-4594-9637-F964693ADFE5
      assertions:
        view: true
```